source/opt/ir_context.h

```diff
@@ -239,6 +263,14 @@ class IRContext {
   // Kill all name and decorate ops targeting the result id of |inst|.
   void KillNamesAndDecorates(ir::Instruction* inst);
 
+  // Return the next unique id for use by an instruction
```

Comment:
While we're nitpicky, prefer "Returns" over "Return".
See style guide:  https://google.github.io/styleguide/cppguide.html#Function_Comments 
In particular:  "These comments should be descriptive ("Opens the file") rather than imperative ("Open the file"); the comment describes the function, it does not tell the function what to do"

Get used to writing comments in that style.  Yes, the codebase is inconsistent on this, but I'm holding Googlers to the Google style guide for this rule.